features/20-exchange-access-token.feature

```diff
@@ -0,0 +1,61 @@
+@smart
+Feature: App exchanges authorization code for access token
+
+    Scenario: Success response has all required parameters
+        Given OAuth is enabled
+        And I am not logged in
+        When I ask for authorization
+        And I exchange my grant code
+        Then the response code should be 200
+        And the JSON response will contain access_token
+        And the JSON response will contain token_type
+        And the JSON response will contain scope
+        And the JSON response will contain state
```

Comment:
Er, would help if I was reading the right part of the spec: 

https://tools.ietf.org/html/rfc6749#section-4.1 (specifically the response details at https://tools.ietf.org/html/rfc6749#section-4.1.4)

clearly indicates that with the authorization code flow, the access token response doesn't have a "state" (it doesn't need to, because the prior step, which provides the code, includes this info).